klayout/drc/rule_decks/mcell.drc

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
```

Comment:
nit: shouldn't that be 2023? :)

Reply:
@proppy We could update all the dates later. Is possible if we can have an issue for that and merge those tables?

Reply:
no need for a bug or to update dates later, it'd be just good to do it for future files.